pkg/util/iptables/iptables.go

```diff
@@ -503,6 +518,18 @@ func getIPTablesWaitFlag(vstring string) []string {
 	}
 }
 
+// Checks if iptables-restore has a "wait" flag
+func getIPTablesRestoreWaitFlag(exec utilexec.Interface) restoreWaitFlag {
+	glog.V(4).Infof("checking if iptables-restore can wait")
+	bytes, err := exec.Command(cmdIPTablesRestore, "--help").CombinedOutput()
```

Comment:
@thockin it's not in a released version yet, and we're sure as hell gonna backport to RHEL